pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -23,31 +23,61 @@ let version = "0.9.6";
         libsmbios
         fwupdate
         libgudev
+        gnutls
         "$out"
       ];
 in
   stdenv.mkDerivation
     { name = "fwupd-${version}";
       src = fetchurl
         { url = "https://people.freedesktop.org/~hughsient/releases/fwupd-${version}.tar.xz";
-          sha256 = "0h3y4ygckvkjdx7yxwbm273iv84yk37ivlcf4xvq95g64vs8gfhf";
+          sha256 = "1im9hih18m0byb1c97r5xds64w21dz6dz9xjxgksnrrm7jkm3bm3";
         };
+
+      nativeBuildInputs = [ meson ninja gtk_doc pkgconfig gobjectIntrospection intltool glibcLocales
+                            valgrind gcab docbook2x libxslt pygobject3 python3Packages.pycairo wrapGAppsHook ];
       buildInputs =
-        [ gtk_doc pkgconfig gobjectIntrospection intltool libgudev
-          polkit appstream-glib gusb sqlite libarchive libsoup
-          docbook2x libxslt libelf libsmbios fwupdate libyaml valgrind
-          meson gpgme libuuid pygobject3 pillow ninja gcab makeWrapper
+        [ polkit appstream-glib gusb sqlite libarchive libsoup
+          libelf libsmbios fwupdate libyaml libgudev
+          gpgme libuuid pillow gnutls glib_networking
         ];
+
+      LC_ALL = "en_US.UTF-8"; # For po/make-images
+
+      NIX_CFLAGS_COMPILE = [
+        "-I${efivar}/include/efivar"
+        # warning: "__LIBELF_INTERNAL__" is not defined
+        "-Wno-error=undef"
+      ];
+
       patches = [ ./fix-missing-deps.patch ];
+      postPatch = ''
+        patchShebangs .
+      '';
+
+      mesonFlags = [
+        "-Denable-colorhug=false"
+        "-Denable-man=false"
+        "-Denable-tests=false"
```

Comment:
Any specific reason to disable tests?

Reply:
Just porting the `configureFlags` from before. I will try to make the tests run later but first we need to make `/var` and `/etc` work.